### PR TITLE
Async stats api requests

### PR DESCRIPTION
**Pull request by rj-lee**
Screenshot 2023-08-26 at 13 58 17

### CODE REVIEW

File: myning/api/players.py
Comment (TheRedPanda17) [inline myning/api/players.py:1]: Why rename?

Comment (rj-lee) [reply myning/api/players.py:1]: I was going with the api endpoint being `players`, do you think changing back to `player` is more appropriate?

Comment (TheRedPanda17) [reply myning/api/players.py:1]: Probably doesn't matter

**Review by TheRedPanda17 — APPROVED** [review]
As long as this has been tested throughly, it looks good to me.

**Comment by rj-lee** [comment]
Yeah, it works

**Pull request merged**